app/collins/util/config/Feature.scala

```diff
@@ -30,6 +30,7 @@ object Feature extends Configurable {
   def encryptedTags = getStringSet("encryptedTags")
   def keepSomeMetaOnRepurpose = getStringSet("keepSomeMetaOnRepurpose", Set())
   def intakeSupported = getBoolean("intakeSupported", true)
+  def intakeIpmiOptional = getBoolean("intakeIpmiOptional", false)
```

Comment:
How about this? I'm still pushing the gh-pages changes. Unfortunately that branch is huge.. :-/ Another TODO I guess..

For now, what about these doc changes?

```
commit 6c03d6e7e6ac0b0ea6a2c0f18168a5ee796ec5af
Author: Johannes 'fish' Ziemke <github@freigeist.org>
Date:   Fri Dec 2 19:06:50 2016 +0100

    Document intakeIpmiOptional

diff --git a/_includes/configuration/features.html b/_includes/configuration/features.html
index 446a041..5cf2bdd 100644
--- a/_includes/configuration/features.html
+++ b/_includes/configuration/features.html
@@ -72,6 +72,11 @@ feature flags are documented below.
       <td>Whether the web based intake is supported or not</td>
     </tr>
     <tr>
+      <td class="inlinecode">intakeIpmiOptional</td>
+      <td>Boolean</td>
+      <td>Whether the tumblr intake process may skip UID verification for assets without IPMI configuration.</td>
+    </tr>
+    <tr>
       <td class="inlinecode">keepSomeMetaOnRepurpose</td>
       <td>Set[String]</td>
       <td>If useWhitelistOnRepurpose is true, this is a list of attributes to keep</td>
```